app/components/custom-source-properties/StreamlabelProperties.vue.ts

```diff
@@ -77,22 +77,20 @@ export default class StreamlabelProperties extends Vue {
     }
 
     this.streamlabelsService.setSettingsForStat(
-      this.currentlySelected.settings.settingsStat ?
-        this.currentlySelected.settings.settingsStat :
-        this.currentlySelected.name,
-      this.labelSettings
+      this.currentlySelected.settings.settingsStat
+        ? this.currentlySelected.settings.settingsStat
```

Comment:
I generally also tend to leave operators dangling on the previous line rather than leading the next line.  That said, I think a lot of the javascript community prefers the latter.  Ultimately this isn't one I feel very strongly about but lets see what the rest of the team things.

Reply:
Hey, so... ternaries, I agree with you all, it's a non-issue if it's just a single line. When it's multiple though, I believe having it at the beginning of next line, makes it obvious the expression isn't done (otherwise you'd have to scan back to the EOL after you look at the next one and you're like "this ain't valid javascript"), also there are some examples from AirBnB here: https://github.com/airbnb/javascript#comparison--nested-ternaries